docs/guide/topics/threads.rst

```diff
@@ -0,0 +1,245 @@
+.. currentmodule:: discord
+
+.. _guide_threads:
+
+Threads
+========
+
+If you've been on a public Discord for a while, you've probably seen a :ddocs:`topics/threads` or two in your time.
+
+They act as offshoots of regular text channels, allowing for a focused, but temporary, location to talk about a given topic.
+
+This section will show you how to create and manage threads within your code.
+
+You can view the API Reference at :class:`Thread`.
+
+Creating a thread
+~~~~~~~~~~~~~~~~~~
+
+There are two types of threads: **public** and **private** threads.
+
+Public threads can be created in any channel where you have the :attr:`~Permissions.create_public_threads` permission.
+
+Private threads, however, require the :attr:`~Permissions.create_private_threads` permission.
+
+Users with the :attr:`~Permissions.manage_threads` permission can see all threads, including private ones they are not explicitly invited to.
+
+You can view the API reference for creating threads here: :meth:`TextChannel.create_thread` / :meth:`Message.create_thread`.
+
+For a quick primer, let's try making a command that creates a thread and then sends a message in it:
+
+.. code-block:: python3
+
+    @bot.command()
+    async def create_thread(ctx: commands.Context):
+        thread = await ctx.channel.create_thread(name="My cool thread", message=ctx.message)
+
+        await thread.send("Created!")
+
+which will create a thread like so:
+
+.. image:: /images/guide/threads/create_public_thread.png
+
+Simple, right? Let's discuss some of the finer details.
+
+Public threads
+~~~~~~~~~~~~~~~
+
+Public threads can be viewed by anyone who can see the channel they belong to.
+People can browse and join these threads as they please.
+
+All public threads are tied to a specific message. You can use a message that already exists, but you can also send a new message to attach the thread to.
```

Comment:
They are not necessarily tied to a specific message.
Public threads can be created without spawning from a message.
![image](https://github.com/Rapptz/discord.py/assets/16943959/9ea049b1-8eef-4f52-b8f9-8e948ffcd419)

This is also possible in the UI
![image](https://github.com/Rapptz/discord.py/assets/16943959/11bf6fa3-afc4-43ea-b71e-e63561ebc259)
![image](https://github.com/Rapptz/discord.py/assets/16943959/47d768ba-d403-4481-a73a-34166ff3f659)